components/service/experiments/src/test/java/mozilla/components/service/experiments/KintoExperimentSourceTest.kt

```diff
@@ -99,37 +173,64 @@ class KintoExperimentSourceTest {
     fun getExperimentsDiffDelete() {
         val httpClient = mock(Client::class.java)
 
-        val storageExperiment = Experiment("id",
-            "name",
-            "description",
-            Experiment.Matcher("eng", "appId", listOf("US")),
-            Experiment.Bucket(10, 5),
-            1523549890000
+        val storageExperiment = testDefaultExperiment(
+            id = "id",
+            description = "description",
+            match = testDefaultMatcher(
+                localeLanguage = "eng",
+                appId = "appId",
+                regions = listOf("US")
+            ),
+            buckets = Experiment.Buckets(5, 5),
+            lastModified = 1523549890000
```

Comment:
nit: magic number, as above (if it's significant we know what the date is)

Reply:
Let's clean those up in a follow-up. Those are not new but rather carried over from the previous test code.